spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -327,7 +327,7 @@ public void testDeleteWithInAndNotInConditions() {
 
     sql("DELETE FROM %s WHERE id NOT IN (1, 10)", tableName);
     assertEquals("Should have expected rows",
-        ImmutableList.of(),
+        ImmutableList.of(row(null, "hr")),
```

Comment:
I cross-checked the output with Postgres.

Reply:
@anjalinorwood, does this look right to you?

Reply:
I talked with @anjalinorwood directly and I think this is correct.